spec/defines/sensu_handler_spec.rb

```diff
@@ -177,4 +177,14 @@
     it { should contain_sensu_handler('myhandler').with_handle_flapping( true ) }
   end
 
+  context 'handle_silenced' do
+    let(:params) { { :command => 'mycommand.rb', :type => 'pipe', :handle_silenced => true } }
+    it { should contain_sensu_handler('myhandler').with_handle_silenced( true ) }
+  end
+
+  context 'handle_silenced set to default' do
```

Comment:
this is the default behavior, which is great. We also need to explicitly test for setting it to false like you do with the test above, setting it to true.